MediaBrowser.Common/Json/JsonDefaults.cs

```diff
@@ -23,6 +23,7 @@ public static JsonSerializerOptions GetOptions()
 
             options.Converters.Add(new JsonGuidConverter());
             options.Converters.Add(new JsonStringEnumConverter());
+            options.Converters.Add(new JsonNonStringKeyDictionaryConverterFactory());
```

Comment:
I tested this PR in combination with #2967 and it did not fix the serialization issue.

I think these defaults are not registered with ASP. You probably need to register here: https://github.com/jellyfin/jellyfin/blob/api-migration/Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs#L87

Reply:
This also depends on https://github.com/jellyfin/jellyfin/pull/3153